frontend/src/components/TicketList.js

```diff
@@ -1,11 +1,26 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { useNavigate } from 'react-router-dom'
-import { Card, CardBody, Text, VStack } from '@chakra-ui/react'
+import { Box, Card, CardBody, Text, VStack } from '@chakra-ui/react'
+import FilterDropdown from './FilterDropdown'
 
 const TicketList = (props) => {
     const navigate = useNavigate()
 
-    const allTickets = Object.entries(props.allTickets)
+    const TICKET_TYPES = Object.freeze({
```

Comment:
same comment as above, can just import, we shouldn't repeat ourselves if not necessary